### PR TITLE
grafana: supply the explanation on the metrics of Flow-Control in TiKV.

### PR DESCRIPTION
**Pull request by qiancai**
This pr supplements the explanation on the Flow-Control metrics in Tikv-Details part.

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/15670
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: grafana-tikv-dashboard.md [hunk 1]
```diff
@@ -185,6 +185,21 @@ This section provides a detailed description of these key metrics on the **TiKV-
 
 ![TiKV Dashboard - Storage metrics](/media/tikv-dashboard-storage.png)
 
+### Flow Control
+
+- Scheduler flow: The scheduler traffic on each TiKV instance in real time.
+- Scheduler discard ratio: The rejection ratio of scheduler requests on each TiKV instance. If this ratio is not 0, it indicates that flow control exists. When `Compaction pending bytes` exceed its threshold, TiKV linearly increases the `Scheduler Discard Ratio` based on the exceeded portion. The client will retry rejected requests automatically.
+- Throttle duration: The duration that the execution of the scheduler requests is blocked when flow control is triggered due to too many L0 files. If this metric has values, it indicates that flow control exists.
```
Comment (qiancai) [inline grafana-tikv-dashboard.md:192]:
```suggestion
- Throttle duration: The block duration for the execution of the scheduler requests when flow control is triggered due to too many L0 files. If this metric has values, it indicates that flow control exists.
```

File: grafana-tikv-dashboard.md [hunk 2]
```diff
@@ -185,6 +185,21 @@ This section provides a detailed description of these key metrics on the **TiKV-
 
 ![TiKV Dashboard - Storage metrics](/media/tikv-dashboard-storage.png)
 
+### Flow Control
+
+- Scheduler flow: The scheduler traffic on each TiKV instance in real time.
+- Scheduler discard ratio: The rejection ratio of scheduler requests on each TiKV instance. If this ratio is not 0, it indicates that flow control exists. When `Compaction pending bytes` exceed its threshold, TiKV linearly increases the `Scheduler Discard Ratio` based on the exceeded portion. The client will retry rejected requests automatically.
```
Comment (LykxSassinator) [inline grafana-tikv-dashboard.md:191]:
```suggestion
- Scheduler discard ratio: The rejection ratio of scheduler requests on each TiKV instance. If this ratio > 0, it indicates that flow control exists. When `Compaction pending bytes` exceeds its threshold, TiKV will linearly increase the `Scheduler Discard Ratio` based on the exceeded portion. The client will retry the rejected requests automatically.
```

File: grafana-tikv-dashboard.md [hunk 3]
```diff
@@ -185,6 +185,21 @@ This section provides a detailed description of these key metrics on the **TiKV-
...
+- Throttle duration: The block duration for the execution of the scheduler requests when flow control is triggered due to too many L0 files. If this metric has values, it indicates that flow control exists.
```
Comment (LykxSassinator) [inline grafana-tikv-dashboard.md:192]:
```suggestion
- Throttle duration: The blocked duration for the execution of the scheduler requests when flow control is triggered due to too many L0 files. If this metric has values, it indicates that flow control exists.
```

File: grafana-tikv-dashboard.md [hunk 4]
```diff
@@ -185,6 +185,21 @@ This section provides a detailed description of these key metrics on the **TiKV-
...
+- Scheduler throttled CF: The CF that triggers RocksDB throttling when the flow control threshold is reached
+- Flow controller actions: The actions that trigger RocksDB throttling when the flow control threshold is reached
+- Flush/L0 flow: The traffic of flush and L0 compaction for different CFs of RocksDB on each TiKV instance
+- Flow control factors: The factors related to triggering RocksDB throttling.
+- Compaction pending bytes: The size of the RocksDB data awaiting compaction in real time on each TiKV instance.
+- Txn command throttled duration: The block duration for commands related to transactions due to throttling. Under normal circumstances, this metric is 0.
```
Comment (LykxSassinator) [inline grafana-tikv-dashboard.md:198]:
```suggestion
- Txn command throttled duration: The blocked duration for commands related to transactions due to throttling. Under normal circumstances, this metric is 0.
```

File: grafana-tikv-dashboard.md [hunk 5]
```diff
@@ -185,6 +185,21 @@ This section provides a detailed description of these key metrics on the **TiKV-
...
+- Non-txn command throttled duration: The block duration for other commands due to throttling. Under normal circumstances, this metric is 0.
```
Comment (LykxSassinator) [inline grafana-tikv-dashboard.md:199]:
```suggestion
- Non-txn command throttled duration: The blocked duration for other commands due to throttling. Under normal circumstances, this metric is 0.
```

File: grafana-tikv-dashboard.md [hunk 6]
```diff
@@ -185,6 +185,21 @@ This section provides a detailed description of these key metrics on the **TiKV-
...
 ![TiKV Dashboard - Storage metrics](/media/tikv-dashboard-storage.png)
 
+### Flow Control
+
+- Scheduler flow: The scheduler traffic on each TiKV instance in real time.
+- Scheduler discard ratio: The rejection ratio of scheduler requests on each TiKV instance. If this ratio is not 0, it indicates that flow control exists. When `Compaction pending bytes` exceed its threshold, TiKV linearly increases the `Scheduler Discard Ratio` based on the exceeded portion. The client will retry rejected requests automatically.
+- Throttle duration: The blocked duration for the execution of the scheduler requests when flow control is triggered due to too many L0 files. If this metric has values, it indicates that flow control exists.
+- Scheduler throttled CF: The CF that triggers RocksDB throttling when the flow control threshold is reached
+- Flow controller actions: The actions that trigger RocksDB throttling when the flow control threshold is reached
+- Flush/L0 flow: The traffic of flush and L0 compaction for different CFs of RocksDB on each TiKV instance
```
Comment (Oreoxmt) [inline grafana-tikv-dashboard.md:195]:
```suggestion
- Scheduler throttled CF: The CF that triggers RocksDB throttling when the flow control threshold is reached.
- Flow controller actions: The actions that trigger RocksDB throttling when the flow control threshold is reached.
- Flush/L0 flow: The traffic of flush and L0 compaction for different CFs of RocksDB on each TiKV instance.
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Review by LykxSassinator — APPROVED** [review]

File: grafana-tikv-dashboard.md [hunk 7]
```diff
@@ -185,6 +185,21 @@ This section provides a detailed description of these key metrics on the **TiKV-
...
+- Throttle duration: The blocked duration for the execution of the scheduler requests when flow control is triggered due to too many L0 files. If this metric has values, it indicates that flow control exists.
+- Scheduler throttled CF: The CF that triggers RocksDB throttling when the flow control threshold is reached.
+- Flow controller actions: The actions that trigger RocksDB throttling when the flow control threshold is reached.
+- Flush/L0 flow: The traffic of flush and L0 compaction for different CFs of RocksDB on each TiKV instance.
+- Flow control factors: The factors related to triggering RocksDB throttling.
+- Compaction pending bytes: The size of the RocksDB data awaiting compaction in real time on each TiKV instance.
+- Txn command throttled duration: The blocked duration for commands related to transactions due to throttling. Under normal circumstances, this metric is 0.
+- Non-txn command throttled duration: The blocked duration for other commands due to throttling. Under normal circumstances, this metric is 0.
```
Comment (hfxsd) [inline grafana-tikv-dashboard.md:199]: 最后这两个，截图中没有

File: grafana-tikv-dashboard.md [hunk 8]
```diff
@@ -185,6 +185,21 @@ This section provides a detailed description of these key metrics on the **TiKV-
 
 ![TiKV Dashboard - Storage metrics](/media/tikv-dashboard-storage.png)
 
+### Flow Control
+
+- Scheduler flow: The scheduler traffic on each TiKV instance in real time.
+- Scheduler discard ratio: The rejection ratio of scheduler requests on each TiKV instance. If this ratio is greater than 0, it indicates that flow control exists. When `Compaction pending bytes` exceeds its threshold, TiKV will linearly increase the `Scheduler Discard Ratio` based on the exceeded portion. The client will retry the rejected requests automatically.
```
Comment (hfxsd) [inline grafana-tikv-dashboard.md:191]:
```suggestion
- Scheduler discard ratio: The rejection ratio of scheduler requests on each TiKV instance. If this ratio is greater than 0, it indicates that flow control exists. When `Compaction pending bytes` exceeds its threshold, TiKV will linearly increase the `Scheduler discard ratio` based on the exceeded portion. The client will retry the rejected requests automatically.
```

Comment (qiancai) [reply grafana-tikv-dashboard.md:191]: updated in https://github.com/pingcap/docs/pull/15765/files

File: grafana-tikv-dashboard.md (hunk 7 above)
Comment (qiancai) [reply grafana-tikv-dashboard.md:199]: updated it in https://github.com/pingcap/docs/pull/15765/files

**Pull request merged**